zkevm-circuits/src/copy_circuit.rs

```diff
@@ -418,20 +418,22 @@ impl<F: Field> SubCircuitConfig<F> for CopyCircuitConfig<F> {
             });
 
             // The first word must not be completely masked.
-            cb.condition(is_word_end.expr(), |cb| {
+            // LOG has no front mask at all.
+            let is_tx_log = meta.query_advice(is_tx_log, Rotation::cur());
+            cb.condition(is_word_end.expr() + is_tx_log.expr(), |cb| {
                 // The first 31 bytes may be front_mask, but not the last byte of the first word.
                 cb.require_zero("front_mask = 0 by the end of the first word", front_mask.expr());
-
-                /* Note: other words may be completely masked, because reader and writer may have different word counts. A fully masked word is a no-op, not contributing to value_acc, and its word_rlc equals word_rlc_prev.
-                cb.require_zero(
-                    "back_mask=0 at the start of the next word",
-                    and::expr([
-                        is_continue.expr(),
-                        back_mask_next.expr(),
-                    ]),
-                );*/
             });
 
+            /* Note: other words may be completely masked, because reader and writer may have different word counts. A fully masked word is a no-op, not contributing to value_acc, and its word_rlc equals word_rlc_prev.
```

Comment:
I think we don't have full mask word, if copy len is zero. no any mask.

Reply:
Here is an example:
- Source range [31..33], requires two words.
- Destination range [0..2], one word + one fully-masked word.

Reply:
I misunderstood all words are masked.